api/API.md

```diff
@@ -221,81 +220,30 @@ Response:
 The response mime type MUST be "application/x-proto-binary".
 The response MUST contain a message with a list of certificates Controller is using. The body MUST be a protobuf message of type [certs.ZControllerCert](./proto/certs/certs.proto).
 
-### Edge Node Certificates
+### Attestation
 
-Publish certificates that edge device would use for signing or any key exchanges. Edge node can include one or more certificates in the request, and include information about each certificate such as unique identifier for the certificate, the target usage for the certificate and any other properties related to the certificate. Each certificate MUST be a X.509 certificate in PEM format. If `isIMutable` property is not set, Controller should reject any request that would overwrite an existing certificate previously published.
+This API anchors all operations related to attestation and trust of an EVE node
 
-   POST /api/v1/edgeDevice/EdgeNodeCerts
+POST /api/v2/edgeDevice/id/<uuid>/attest
```

Comment:
There is some markdown issue with the <uuid> because it doesn't display in https://github.com/cshari-zededa/eve/blob/attest-api/api/API.md

Reply:
Addressed. There is an issue with "<uuid>". I changed it to {uuid} to align with the style in other parts of API.md where URL qualifiers are mentioned.